### ISSUE TITLE
508-defect-2 [SCREENREADER]: "Link copied" announcement is spotty on VO/Safari

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#25275]
# [508-defect-2 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I want to know if the link has been copied to have an equivalent experience to sighted users.

## Details
"Link copied" will sometimes fail to be announced to screen reader users.

https://user-images.githubusercontent.com/14154792/119681293-f0487780-be0f-11eb-8ebd-75d5e09f99f2.mov

## Acceptance Criteria
- [ ] Screen reader users can consistently know when a link has been copied

## Steps to Recreate

1. Turn on VoiceOver on Safari
2. Go through the page and copy several links
3. Confirm "link copied" is occasionally unannounced and is inconsistent

## Proposed Solution 

We could [pursue a multi-function button](https://adrianroselli.com/2021/01/multi-function-button.html) if we're willing to give the button a label.
![image](https://user-images.githubusercontent.com/14154792/119682903-40740980-be11-11eb-8586-62662a792fb9.png)

Otherwise, we'll need to do more digging into this to find the right solution.
